source/manage/mmctl-command-line-tool.rst

```diff
@@ -130,7 +130,7 @@ Install mmctl
 
 |all-plans| |cloud| |self-hosted|
 
-There are different methods available to install mmctl.
+The mmctl tool is installed with Mattermost from v6.0. For customers using Mattermost prior to v6.0, there are different methods available to install mmctl.
```

Comment:
`mmctl` is interesting as well for other use cases than versions previous to `v6`, say a sysadmin wanting to manage a Mattermost instance from their laptop, so we could say something like:

```suggestion
The mmctl tool comes bundled with the Mattermost package from v6.0. For customers that want to setup it independently from the package, or for versions prior to v6.0, there are different methods available to install mmctl.
```